MAUI/DataForm/layout.md

```diff
@@ -0,0 +1,750 @@
+
+{% tabs %}
+{% highlight xaml %}
+
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:DataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+             x:Class="CommitMode.MainPage">
+
+    <DataForm:SfDataForm
+        x:Name="DataForm"
+        DataObject="{Binding DataObjects}"
+        GenerateDataFormItem="OnGenerateDataFormItem">
+    </DataForm:SfDataForm>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# %}
+
+this.DataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+        {
+            e.DataFormItem.DefaultLayoutSettings = new DataFormDefaultLayoutSettings
+            {
+                LabelPosition = DataFormLabelPosition.Left,
+            };
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Loading images for label
+
+Image can be loaded instead of label  by using [LeadingLabelIcon] property of `DataFormItem`.
+
+## Using Event
+
+{% tabs %}
+{% highlight c# %}
+
+this.DataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "Name")
+            {
+                e.DataFormItem.LeadingLabelIcon = ImageSource.FromFile("ContactInfo.png");
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Changing order of the DataFormItem
+
+The order of the `DataFormItem` can be changed by using attributes or by handling `GenerateDataFormItem` event.
+
+### Using attributes
+
+The fields order can be changed by using the `RowOrder` property in [DataFormDisplayOptionsAttribute] attribute.
+
+{% tabs %}
+{% highlight c# %}
+
+    [DataFormDisplayOptions(RowOrder = 1)]
+    public string Adress { get; set; }
+
+    [DataFormDisplayOptions(RowOrder = 0)]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+### Using event
+
+The fields order can be changed by using the [RowOrder] property in the `DataFormItem`.
+
+{% tabs %}
+{% highlight c# %}
+
+this.DataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "Name")
+            {
+                e.DataFormItem.RowOrder = 0;
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Changing order of the DataFormItem in same row
+
+The order of the `DataFormItem` can be changed within a row by using attributes or by handling `GenerateDataFormItem` event.
+
+### Using attributes
```

Comment:
#### Using attributes

The order of the editors in grid rows can be changed by using the `ItemsOrderInRow` property in [DataFormDisplayOptionsAttribute] attribute. 

#### Using event

The order of the editors in grid rows can be changed by using the [ItemsOrderInRow] property in the `DataFormItem`

Reply:
changed content